hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/metrics/MetricsContext.java

```diff
@@ -208,4 +213,51 @@ public long get() {
             return VOLATILE_VALUE_UPDATER.get(this);
         }
     }
+
+    private static final class DummyMetric implements Metric {
```

Comment:
Better name might be `NoopMetric`. We can even use singleton instance of this class, it's stateless.

Reply:
Renamed it, but making it a singleton is not really an option until we have the `name` and `unit` methods `Metric`.

Reply:
True, I retract